src/App.js

```diff
@@ -66,17 +66,48 @@ class InterviewerShift extends React.Component {
         return filteredCalls;
     }
 
-    claimBelongsInHour(claim, hour) {
-        // if the claim starts inside the given hour, include it
-        if (claim.starttime.startsWith(hour + ':')) {
+    processJSON() {
+        this.props.shift.hours.forEach( hour => {
+            this.props.shift.calls.forEach( call => {
+                if ( 
+                    this.belongsInHour(call, hour.hour) && 
+                    call.outcomecategory !== outcomeCategories.WHITE_SPOT 
+                ) {
+                    hour["hasAttempt"]  = true;
+                }
```

Comment:
loop could use a break here, once you've found one you're good.
after looking a bit, this could be useful:
https://www.w3schools.com/jsref/jsref_some.asp

Reply:
Ah yes! haha I meant to do the break, but forgot :P thanks!